addon/services/airbrake.js

```diff
@@ -3,7 +3,8 @@ import getClient from '../utils/get-client';
 import setEnvironment from '../filters/environment';
 import loggerReporter from '../reporters/logger';
 import setSession from '../filters/session';
-import getOwner from 'ember-getowner-polyfill';
```

Comment:
@jeffreyjurgajtis hm, doesn't seem like the polyfill is being added in ember 1.13.13. investigating.